packages/cli/src/utils/checks.ts

```diff
@@ -295,6 +295,16 @@ class CheckBuilder {
     )
   }
 
+  hasEnoughUsd = (account: Address, value: BigNumber) => {
+    const valueInEth = this.kit.web3.utils.fromWei(value.toFixed(), 'ether')
+    return this.addCheck(`Account has at least ${valueInEth} cUSD`, () =>
+      this.kit.contracts
+        .getStableToken()
+        .then((gt) => gt.balanceOf(account))
```

Comment:
Should this be `st`? I personally would even prefer not abbreviating here or above, `stableToken` and `goldToken`.

Reply:
Yeah, decided to replace other abbreviations from this file, too.